sonar-scanner-engine/src/main/java/org/sonar/scanner/scan/ModuleScanContainer.java

```diff
@@ -177,12 +172,12 @@ private void addExtensions() {
   @Override
   protected void doAfterStart() {
     DefaultIndex index = getComponentByType(DefaultIndex.class);
-    index.setCurrentProject(module, getComponentByType(DefaultSensorStorage.class));
+    index.setCurrentProject(getComponentByType(DefaultSensorStorage.class));
 
     getComponentByType(AbstractPhaseExecutor.class).execute(module);
 
     // Free memory since module settings are no more used
-    module.setSettings(null);
+    //module.setSettings(null);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)